fbw-common/src/wasm/systems/systems/src/simulation/update_context.rs

```diff
@@ -306,6 +308,7 @@ impl UpdateContext {
     pub(crate) const LOCAL_VERTICAL_SPEED_KEY: &'static str = "VELOCITY BODY Y";
     pub(crate) const ALT_ABOVE_GROUND_KEY: &'static str = "PLANE ALT ABOVE GROUND";
     pub(crate) const LATITUDE_KEY: &'static str = "PLANE LATITUDE";
+    pub(crate) const ANGLE_OF_ATTACK_KEY: &'static str = "ANGLE OF ATTACK INDICATOR";
```

Comment:
We should not use the "INDICATOR" simvars at all, they are meant for cockpit gauges. This one is definitely not the actual AOA, but instead AOA as a ratio of stall AOA for one of these type of indicators:
![garmin-aoa-small](https://github.com/flybywiresim/aircraft/assets/9995998/76a7d83d-d92d-4a06-b01b-84d8ba0e207f)

Any aircraft systems that consume AOA almost definitely consume the ADR AOA words from their ARINC input bus anyway. No other system can read the AOA vanes directly.

Reply:
I'm now reading in the ADR AOA value